fixtures/tracks/animal/exercises/TRACK_HINTS.md

```diff
@@ -0,0 +1 @@
+This is the content of the track hints file
```

Comment:
Trackler currently supports tracks that meet one of the two following:

* have an `exercises` directory, and **all** their exercises in it.
* do not have an `exercises` directory at all.

The question is whether it should support tracks that have an `exercises` directory, but **not all exercises are in it**.